roles/ovirt-storages/README.md

```diff
@@ -1,7 +1,7 @@
-oVirt storages
+oVirt Storages
 ==============
```

Comment:
I remember we were talking about it, and sticked to `ovirt-storages`, but I don't remember why.